workflows/qc/quality-check-standard.wdl

```diff
@@ -97,12 +103,24 @@ workflow quality_check {
     call ngsderive.encoding as ngsderive_encoding { input: ngs_files=[quickcheck.checked_bam], prefix=prefix, max_retries=max_retries }
     String parsed_encoding = read_string(ngsderive_encoding.inferred_encoding)
 
-    if (experiment == "WGS") {
-        call picard.collect_wgs_metrics { input: bam=quickcheck.checked_bam, reference_fasta=reference_fasta, max_retries=max_retries }
-        call mosdepth.coverage { input: bam=quickcheck.checked_bam, bai=bam_index, max_retries=max_retries }
+    call mosdepth.coverage as wg_coverage {
+        input:
+            bam=quickcheck.checked_bam,
```

Comment:
Does this need to be `quickcheck.checked_bam` or should it just be the workflow input? I suppose `quickcheck` is quick and might help cut down on wasted compute. But we don't necessarily need to wait for quickcheck to complete to start queueing the remaining tasks.

Reply:
The rationale (which I'm willing to reconsider):
`quickcheck` is "quick", so we should validate the input BAM before queuing long running (and potentially expensive) tasks that would fail if `quickcheck` failed. I don't think overall runtime is a valid concern with how fast quickcheck is. So I'd like to keep it this way. Unless you have some concern about readability or something else I haven't thought of (maybe this decision warrants a comment somewhere in the code?)